docs/content/Examples-Tutorials-Recipes/Recipes/Access-control/controlling-access-to-cubes-and-views.mdx

```diff
@@ -30,35 +30,63 @@ module.exports = {
 
 ## Data modeling
 
+<CodeTabs>
+
+```yaml
+# Orders.yml
```

Comment:
I think this and two other file names should match actual cube names (with extension added, obvi).